### DISCUSSION TITLE
Can you have "File annotations" set to "always on", or a command for "Toggle File Changes"?

### DISCUSSION THREAD

**Discussion by ghost** [gitkraken/vscode-gitlens#2128] [Q&A] [answered]
Can you have "File annotations" set to "always on", or a command for "Toggle File Changes"? 

For example, its a great way to visualize see code changes while you work. But having to toggle it on for each file seems a 
little tedious. Can you have it set always on on a specific folder or file type? or just on for every file? That would be awesome. 

Example where it is great to have.

![image](https://user-images.githubusercontent.com/20955623/181516858-c4d49887-4ae5-4b09-9318-9a0f9e15c1cf.png)

**Comment by eamodio** [comment, marked as answer]
You can change the default toggle behavior to be for all files instead of for each file individually:

![image](https://user-images.githubusercontent.com/641685/181583360-8d911b2f-3727-42da-845d-ef2676263b9e.png)

And from the Command Palette you can run the toggle command -- and if you click the gear on the right side you can setup your own keybinding for it:

![image](https://user-images.githubusercontent.com/641685/181583619-e6ccf3a9-f912-405c-9c4c-2edcc477b2d9.png)

Hope that helps!

**Comment by AdmiralRazorbeard** [comment]
This is pretty cool, is there a way to change the highlights color of the lines changes? I'd like to reduce their intensity to be more subtle. I would love to have that option activated that way all the time

Also an option to restrict it only to local (unpublished) changes would be great. Thanks for your work.
